front/app/modules/commercial/content_builder/admin/components/CraftComponents/Button/index.tsx

```diff
@@ -0,0 +1,218 @@
+import { injectIntl } from 'utils/cl-intl';
+
+type ButtonProps = {
+  text: string;
+  url: string;
+  opensInNewTab: boolean;
+  type: ButtonStyles;
+  alignment: string;
+};
+
+const Button = ({ text, url, opensInNewTab, type, alignment }: ButtonProps) => {
+  const openUrl = (url: string) => {
+    const targetValue = opensInNewTab ? '_blank' : '_self';
+    window.open(url, targetValue);
+  };
+
+  return (
+    <Box
+      display="flex"
+      justifyContent={
+        alignment === 'center'
+          ? 'center'
+          : alignment === 'left'
+          ? 'flex-start'
+          : 'flex-end'
+      }
+    >
+      <ButtonComponent
+        onClick={() => openUrl(url)}
+        id="e2e-button"
+        width={alignment === 'fullWidth' ? '100%' : 'auto'}
+        buttonStyle={type}
+        text={text}
+      />
+    </Box>
+  );
+};
+
+const ButtonSettings = injectIntl(({ intl: { formatMessage } }) => {
+  const {
+    actions: { setProp },
+    text,
+    url,
+    opensInNewTab,
+    type,
+    alignment,
+  } = useNode((node) => ({
+    text: node.data.props.text,
+    url: node.data.props.url,
+    opensInNewTab: node.data.props.opensInNewTab,
+    type: node.data.props.type,
+    alignment: node.data.props.alignment,
+  }));
+  return (
+    <Box background="#ffffff" marginBottom="20px">
+      <Box flex="0 0 100%">
+        <Input
+          id="e2e-button-text-input"
+          label={
+            <Title variant="h4" as="h3">
+              {formatMessage(messages.buttonText)}
+            </Title>
+          }
+          placeholder={formatMessage(messages.button)}
+          type="text"
+          value={text}
+          onChange={(value) => {
+            setProp((props) => (props.text = value));
+          }}
+        />
+      </Box>
+      <Box flex="0 0 100%">
+        <Input
+          id="e2e-button-url-input"
+          label={
+            <Title variant="h4" as="h3">
+              {formatMessage(messages.buttonUrl)}
+            </Title>
+          }
+          placeholder={formatMessage(messages.urlPlaceholder)}
+          type="text"
+          value={url}
+          onChange={(value) => {
+            setProp((props) => (props.url = value));
+          }}
+        />
+      </Box>
+      <Box mt="20px" mb="20px">
+        <Toggle
+          id="toggle-open-new-tab-window"
+          checked={opensInNewTab}
+          onChange={() => {
+            setProp((props) => (props.opensInNewTab = !opensInNewTab));
+          }}
+          label={formatMessage(messages.openInTabOrWindow)}
+        />
+      </Box>
+      <Title variant="h4" as="h3">
+        {formatMessage(messages.buttonTypeRadioLabel)}
+      </Title>
+      <Radio
+        onChange={(value) => {
+          setProp((props) => (props.type = value));
+        }}
+        currentValue={type}
+        id="style-primary"
+        name="buttonStyle"
+        value={'primary'}
+        label={
+          <ButtonComponent
```

Comment:
Could you just use a Box with the appropriate colors here, rather than an actual button - not very semantic or clear to have an unclickable button as the label..

Reply:
Just do "primary" | "secondary" and that's it. :)

Reply:
Ahh ok - thank you!!